sdcm/nemesis.py

```diff
@@ -180,6 +180,23 @@ def disrupt_hard_reboot_node(self):
         self.log.info('Waiting JMX services to start after node reboot')
         self.target_node.wait_jmx_up()
 
+    def disrupt_multiple_hard_reboot_node(self):
+
+        num_of_reboots = random.randint(2, 10)
+        for i in range(num_of_reboots):
```

Comment:
print out `num_of_reboots` and `i`.
For example `Rebooting {i} of {num_of_reboots} times".format(**locals())`